app/components/form_components/autocomplete/sort.js

```diff
@@ -1,15 +1,16 @@
 // Returns an array of integers representing the position of each regex match.
 const matchPositions = (string, regexes) => regexes.map(regex => string.search(regex)).filter(i => i >= 0)
 
-const calculateWeight = (name, rawSynonyms, query) => {
+const calculateWeight = (name, synonyms = [], query) => {
```

Comment:
We need the optional arguments to be at the end, otherwise we don't have a way of just calling calculateWeight with a name and a query.